packages/form_builder_validators/lib/src/form_builder_validators.dart

```diff
@@ -146,6 +146,27 @@ class FormBuilderValidators {
     };
   }
 
+  /// [FormFieldValidator] that requires the length of the field's value to be
+  /// equal to the provided length.
```

Comment:
Add description with which types works (Iterable and String)

Reply:
I added a description.